README.md

```diff
@@ -61,231 +24,142 @@ For more information on the importance of a professional README for your reposit
+<li>Integration</li>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
 
 <details>
-  <summary>Client</summary>
+  <summary>Ruby</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://reactjs.org/">Ruby</a></li>
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>Rails</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://expressjs.com/">Rails</a></li>
   </ul>
 </details>
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+- **[Ruby]**
+- **[Rails]**
+- **[postgtreSQL]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
+Git clone https://github.com/SAZZAD1Q2/BudgetMate.git
+Run in your browser
 
 ### Prerequisites
 
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+To run this project you need:
+Git
+An IDE like vs code . bracket etc.
+Code Editor
+Browser
 
 ### Setup
 
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
+Clone this repository to your desired folder: https://github.com/SAZZAD1Q2/BudgetMate.git
+Opent it to your code editor
+open it then to your browser.
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
 
 ### Install
 
 Install this project with:
+Github or git or vs code. For this open your terminal and proceed some steps:
+a. Install node package using the command : npm install
+b. Then run the website using the command : npm start
+c. Test the code using jest snapShots and for this use the Command : npx jest
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Deployment
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
 
-You can deploy this project using:
+- [Live Demo Link](https://sazzad-3q4t.onrender.com)
```

Comment:
- [ Required ] I think it would be nice if you ensure that the locally deployed project is the same as the one that has been deployed online. Currently, when I try to open the Live demo link I get a black screen that takes too long to load but never loads. Please see the screenshot below and rectify this.

## Screenshot
![Screenshot from 2023-10-21 09-48-02](https://github.com/SAZZAD1Q2/BudgetMate/assets/16392046/17b151f7-1925-4078-a2a1-465c24ab7f18)

Reply:
Dear @Felix45 
Thank you for reviewing my PR. Actually, I am sorry that the project that has been deployed online is not displaying properly as the local display. Though when I diployed it was displayed well. Now I will check it and submit it again for approval.

Reply:
> * [ Required ] I think it would be nice if you ensure that the locally deployed project is the same as the one that has been deployed online. Currently, when I try to open the Live demo link I get a black screen that takes too long to load but never loads. Please see the screenshot below and rectify this.
> 
> ## Screenshot
> ![Screenshot from 2023-10-21 09-48-02](https://user-images.githubusercontent.com/16392046/277093173-17b151f7-1925-4078-a2a1-465c24ab7f18.png)
> 
> ![Screenshot from 2023-10-21 09-58-57](https://user-images.githubusercontent.com/16392046/277093587-af90de3e-2e53-43e5-9849-83f7a0572421.png)

- [ ] Kindly fix the live demo app link to work properly. It is taking forever to open.